pkg/bisect/bisect.go

```diff
@@ -29,13 +34,15 @@ type Config struct {
 }
 
 type KernelConfig struct {
-	Repo      string
-	Branch    string
-	Commit    string
-	Cmdline   string
-	Sysctl    string
-	Config    []byte
-	Userspace string
+	Repo           string
+	Branch         string
+	Commit         string
+	Cmdline        string
+	Sysctl         string
+	Config         []byte
+	BaselineConfig []byte
+	MinConfig      []byte
```

Comment:
I think this be in the output because it would be super useful to attach the minimized config to the bug report.

Reply:
Done